pytest_splunk_addon/splunk.py

```diff
@@ -101,81 +101,36 @@ def pytest_addoption(parser):
             " of 8.0.0. "
         ),
     )
+    group.addoption(
+        "--field-bank",
+        action="store",
+        dest="field_bank",
+        default="",
+        help="Path of the field JSON file. Refer to ReadMe for the Json format.",
+    )
 
 
 @pytest.fixture(scope="session")
-def docker_compose_files(pytestconfig):
+def splunk_search_util(splunk):
```

Comment:
not sure when splunk is passed in here, but I'm not a particular fan of having the "port" for splunkd be called just "port", but if we don't need anything else like the web port then I suppose this is fine.

Reply:
splunk is a fixture used to get the Splunk details through either a docker container or from options. 
1. splunklib.client has a similar parameter "port". 
2. In the current scope, we're only working with the management port.